### ISSUE TITLE
Milestone-1: Evaluation of Tools

### ISSUE THREAD

**Issue by harunerkurt** [bounswe/bounswe2022group1#158]
Issue Description
For the milestone report Evaluation of the tools section should be created.

Step Details
Steps that will be performed:

 Evaluation of Discord
 Evaluation of GitHub
 Evaluation of Canva
 Evaluation of Lucidchart
 Evaluation of Slack
 Evaluation of Whatsapp

Deadline of the Issue
15.04.2022 - 17:00

Deadline for the Review
15.04.2022  - 20:00

**Comment by harunerkurt** [comment]
I created the Evaluation of Tools Section and I added followings:
"4.1. GitHub

We used GitHub as our project version management system. We did some research on github and added this research to our repo. ​​GitHub is a cloud-based service that hosts project code and structure as well as track and control changes to projects' code. GitHub makes controlling codes and project development a lot easier for developers to use Git for version control and collaboration. GitHub helps developers work together on a project from any location while facilitating collaboration. GitHub is a user-friendly git repository. Thus, anyone can use GitHub to do things a developer can do with git commands via the interface.


4.2. Slack

We used slack for communication. Slack is a messaging app for business so We thought that the slack app could help us in business. Slack brings people together to work as one unified team.

4.3. Zoom

During the distance-education, we have used Zoom for our weekly meetings. Zoom was a very convenient application for many people to have meetings at the same time. Breakout rooms allowed us to work in teams. The video quality is well

4.4. LucidChart

LucidChart is a chart creation tool and free for students. But it also has a premium mode and it's paid. We used it for creating our diagrams. It lets multiple users edit diagrams at the same time.

4.5. Canva

We used Canva for creating our mock-ups. It is a free tool and pretty useful. It is also possible to work simultaneously. Also, it's pretty easy to create mockups.

 	4.6. Whatsapp
WhatsApp is currently the most used messaging application and we preferred WhatsApp because everyone has a WhatsApp account."
